web/services/models.py

```diff
@@ -65,7 +65,9 @@ def notify_report_received(sender, instance, signal, *args, **kwargs):
     :param args: Unused
     :param kwargs: Unused
     """
-
+    if instance.user is None:
```

Comment:
Did you want the stack trace in this case?

Reply:
Yes

Reply:
will I need this none protection for instance.stacktrace and instance.textBox?